sqlmesh/dbt/target.py

```diff
@@ -111,20 +111,20 @@ def attribute_dict(self) -> AttributeDict:
         fields["target_name"] = self.name
         return AttributeDict(fields)
 
-    @property
-    def quote_policy(self) -> Policy:
+    @classmethod
```

Comment:
These all can be classproperties: https://github.com/TobikoData/sqlmesh/blob/b62dfab5b6f741486c9b3dbd69b6a3a1dfe6145b/sqlmesh/utils/__init__.py#L237